src/pyfaf/actions/create_problems.py

```diff
@@ -454,11 +454,9 @@ def _create_problems(self, db, problemplugin, #pylint: disable=too-many-statemen
 
             self.log_debug("Removing {0} invalid reports from problems"
                            .format(len(invalid_report_ids_to_clean)))
-            for report_id in invalid_report_ids_to_clean:
-                db_report = get_report_by_id(db, report_id)
-                if db_report is not None:
-                    db_report.problem_id = None
-                    db.session.add(db_report)
+            for db_report in get_reports_for_ids(db, invalid_report_ids_to_clean, 1000):
+                db_report.problem_id = None
```

Comment:
Can this be updated by something like `UPDATE report SET problem_id WHERE id in <IDS>`?

Reply:
Hm, good point. I’ll look into that.